fleetctl/unload.go

```diff
@@ -36,6 +36,8 @@ func init() {
 }
 
 func runUnloadUnit(args []string) (exit int) {
+	attempts := getBlockAttempts()
```

Comment:
Here and elsewhere, no need for a separate variable, just inline it in the tryWaitForUnitStates call